batch/src/main/scala/uk/gov/ons/addressindex/models/CSVSchemas.scala

```diff
@@ -220,4 +220,16 @@ object CSVSchemas {
     StructField("entryDate", DateType, nullable = false),
     StructField("successor", LongType, nullable = false)
   ))
+
+  /**
+    * hierarchy CSV file schema
+    */
+  val hierarchyFileSchema = StructType(Seq(
+    StructField("uprn", LongType, nullable = true),
```

Comment:
Uprn must not be nullable and NOT repeated in the same hierarchy extract - it is the key of the hierarchy